src/main/java/jackyy/dimensionaledibles/block/BlockCakeBase.java

```diff
@@ -282,6 +282,15 @@ protected void consumeCake(World world,
         }
     }
 
+    /**
+     * Get the Cake Fuel as an ItemStack to maintain NBT data and Metadata.
+     * @return The Fuel as an ItemStack
+     */
+    private ItemStack getFuelItemStack() {
+        return new ItemStack(Objects.requireNonNull(
```

Comment:
I'm a bit concerned about the use of `Objects::requireNonNull(Object)` here, as it could throw and crash the game if the configured fuel is not found in the item registry. I think it would be appropriate to use a safer mechanism that is tolerant of a typo in the config file.

Reply:
I think the best way to go about doing this is to have a default value of whatever the cake usually uses as a fuel. For example for overworld cake, it will default to Oak Sapling if the config is malformed

Reply:
We could hardcode some kind of fallback for the Vanilla cakes but that wouldn't really work for the Custom Cake.

Reply:
Im just not sure of how to handle the case otherwise. At some point a null is going to have to be handled, probably as a check in the onBlockActivated, which i guess in that case, we just skip?

Reply:
We can fallback for the Vanilla cakes with a default fuel that gets used in the event of null, but for custom cakes since it's all computed on the fly we have to implement custom logic to work around it, like what you're suggesting.

I can implement the fallback values in the feature branch and we can proceed from there.